### PR TITLE
Added payment classes

### PR DESCRIPTION
**Pull request by SnehaGhadge1**

### CODE REVIEW

File: src/Pages/PaymentPage.java
Comment (SnehaGhadge1) [inline src/Pages/PaymentPage.java:1]: Hi @snehaghadge70 

please review payment page class.

File: src/Tests/PaymentTest.java
Comment (SnehaGhadge1) [inline src/Tests/PaymentTest.java:1]: hi @snehaghadge70 

please review payment Test class.

File: src/Pages/PaymentPage.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package Pages;
+
+public class PaymentPage {
+
+	public void paymentPage() {
+		System.out.println("get the paymnet");
```
Comment (snehaghadge70) [inline src/Pages/PaymentPage.java:6]: hi @SnehaGhadge1 
please add logic methods.

File: src/Tests/PaymentTest.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package Tests;
+
+public class PaymentTest {
+
+	String paymnetOption = "UPI";
+	public static void main(String[] args) {
+		
+
```
Comment (snehaghadge70) [inline src/Tests/PaymentTest.java:8]: @SnehaGhadge1 
please add UPI methods.

Comment (SnehaGhadge1) [reply src/Tests/PaymentTest.java:11]: code changes done for Test class

File: src/Pages/PaymentPage.java (hunk 1 above)
Comment (SnehaGhadge1) [reply src/Pages/PaymentPage.java:7]: code changes done for page class

**Review by snehaghadge70 — APPROVED** [review]
Approved

**Comment by SnehaGhadge1** [comment]
merging paymnetPage changes with remote master branch.

**Pull request merged**